sphinx_search/_static/js/rtd_sphinx_search.js

```diff
@@ -334,17 +334,16 @@ const showSearchModal = () => {
     let search_bar = getInputField();
     search_bar.blur();
 
-    let search_outer_wrapper = document.querySelector(
-        ".search__outer__wrapper"
-    );
-    search_outer_wrapper.classList.add("display-block");
-
-    // sets the value of the input field to empty string and focus it.
-    let search_outer_input = document.querySelector(".search__outer__input");
-    if (search_outer_input !== null) {
-        search_outer_input.value = "";
-        search_outer_input.focus();
-    }
+    $(".search__outer__wrapper").fadeIn(400, () => {
```

Comment:
Did this value come from anywhere? It seems slight long to me but I don't have a particular value in mind.

Reply:
It feels smooth with this.
Should I reduce it to 200 or 100?

Reply:
400 seems fine. Sounds like a thing we'll get feedback on from real users.